### PR TITLE
#2620 split out admin search

### PR DESCRIPTION
**Pull request by stenington**
On the participants page, instead of "Name or Email" you can now search separately by First name, Last name, Email, Parent or guardian name, Parent or guardian email.

### CODE REVIEW

File: app/data_grids/accounts_grid.rb [hunk 1]
```diff
@@ -407,16 +407,77 @@ class AccountsGrid
 
   filter :name_email,
     header: "Name or Email",
```
Comment (shaunxp20) [inline app/data_grids/accounts_grid.rb:409]: Just double checking, were we going to split/remove the "Name or Email" field, or split/keep this field?

**Review by shaunxp20 — APPROVED** [review]
Didn't test it, but the additions looks okay to me! 👍

File: app/data_grids/accounts_grid.rb (hunk 1 above)
Comment (stenington) [reply app/data_grids/accounts_grid.rb:409]: And all the new fields are only shown to admins.

**Comment by stenington** [comment]
Changed this back so both admins and chapter ambassadors have the same fields.

**Review by shaunxp20 — APPROVED** [review]

**Pull request merged**